signup.php

```diff
@@ -48,21 +41,23 @@
 					$Password = $_POST['password'];
 
 				//create a variable to store sql code for the ''And Users' query
-				$insertquery = "INSERT INTO User(User_ID, Password) VALUES ('".$User_ID."', '".$Password."')";
+				$insertquery = "INSERT INTO User(User, Password) VALUES ('".$User_ID."', '".$Password."')";
 				if (mysqli_query($conn,$insertquery))
 					{
-					echo "<p class = 'error'>Record inserted:</p>";
+					echo "<p class = 'Connected'>Record inserted:</p>";
 					}
 				else
 					{
-					echo "<p id = 'error'>Error inserting record:</p>";
+					echo "<p class = 'error'>Error inserting record:</p>";
```

Comment:
04/08/2021: I called the error as class because in the css page I called the selector as .error not #error so it is correct if I called it class.